UML-Diagram/Master-UML.puml

```diff
@@ -48,8 +49,6 @@ package "Library system"{
     StaffMenu ..> ReportStaffMenu
     StaffMenu ..> StaffSearchMenu
     Login ..> StaffMenu
-    ABorrowReport ..* BorrowManager
-    APatronReport ..* PatronManager
```

Comment:
The use of the IReport has we are talking in the conversation it should reference StaffReportMenu, i dont remember if its in use in other classes

Reply:
Now it's with its relationships but there is no a class according to the actual code into **develop** branch, which is `BorrowPatronHistoryInput`. I guess we can add it later, right?

Reply:
Maybe if its needed an update but as i see the only use for Report was there on ReportStaffMenu so yes.